camel/loaders/firecrawl_reader.py

```diff
@@ -144,6 +150,7 @@ def structured_scrape(self, url: str, response_format: BaseModel) -> Dict:
                 {
                     'formats': ['extract'],
                     'extract': {'schema': response_format.model_json_schema()},
+                    'integration': 'camelai',
```

Comment:
Why is this method different? The others check if params is None first, but this one just hardcodes it. 
Also this method doesn't take a params argument like the others do.